cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -4356,7 +4356,7 @@ TEST(Substrait, PlanWithAsOfJoinExtension) {
           "extension_multi": {
             "common": {
               "emit": {
-                "outputMapping": [0, 1, 2, 5]
+                "outputMapping": [0, 1, 2, 3]
```

Comment:
Why these change

Reply:
Before these indices where input indices but now they are output indices. The output schema has these columns: "time", "key", "value1", "value2".